assets/js/base/components/cart-checkout/product-name/index.js

```diff
@@ -9,7 +9,7 @@ import { decodeEntities } from '@wordpress/html-entities';
  */
 import './style.scss';
 
-const ProductName = ( { name, permalink, disabled } ) => {
+const ProductName = ( { name, permalink, disabled = false } ) => {
```

Comment:
Changes in this file are not related to this PR, but linting was complaining about it.